Source/Controllers/DataControllers/SPTableData.m

```diff
@@ -1004,19 +1110,15 @@ - (BOOL)updateStatusInformationForCurrentTable
 
 		// Reassign any "Type" key - for MySQL < 4.1 - to "Engine" for consistency.
```

Comment:
This should be... removed? Or all these workarounds?

Reply:
don't know. doesn't hurt to leave them?